src/services/combineResponses.ts

```diff
@@ -180,10 +179,6 @@ function findSourceField(referenceField: Field, sourceFields: Field[], index: nu
     return candidates[0];
   }
 
```

Comment:
This was taken from the original grafana/grafana implementation, for non-Loki data sources. https://github.com/grafana/grafana/pull/90650